content/recipes/authoring/kata-snippets/submission-tests.md

```diff
@@ -0,0 +1,88 @@
+---
+kind: recipe
+sidebar: authoring
+prev: /recipes/authoring/kata-snippets/sample-tests/
+next: /recipes/authoring/kata-snippets/preloaded/
+---
+
+# Writing Submission Tests
+
+This article contains a set of guidelines for kata authors or translators to create good test suites for their kata. They were collected to help ensure that kata, translations and their test suites are of sufficient quality and users' experience with tests will be as good as possible.
```

Comment:
```suggestion
This article contains a set of guidelines for kata authors or translators to create good test suites for their kata. They were collected to help ensure that kata, translations and their test suites are of sufficient quality so that users' experience with tests will be as good as possible.
```

Reply:
LGTM